recipes/conda/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "4.1.5" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/conda/conda
+  git_tag: {{ version }}
+  git_depth: 1
+
+build:
+  number: 0
+  script:
+    - unlink "${PREFIX}"/bin/conda                  # [unix]
```

Comment:
Yes, this is really a special case *just* for conda.  I burned a whole day figuring that one out.